CONTRIBUTING.md

```diff
@@ -285,47 +295,47 @@ Every other rule serve to complete the module. These other rules can vary consid
 An example rule that follows most of these principles is included below (taken from the `star` module).
 
 1. Each rule should only consist of one command, unless the rule uses standard tools like `gzip` for additional commands. Otherwise, split into multiple rules, optionally connected using `pipe()` or `temp()` to avoid intermediate files.
-   
-   > This guideline ensures that rules are module and can easily be rearranged by the user. It also ensures that tool-specific conda environments (_e.g._ `samtools`, `star`) can be used, which is not possible is more than one tool is used in a rule. 
-   
+
+   > This guideline ensures that rules are module and can easily be rearranged by the user. It also ensures that tool-specific conda environments (_e.g._ `samtools`, `star`) can be used, which is not possible is more than one tool is used in a rule.
```

Comment:
`... rules are module ...`
is this part is a typo?

Reply:
Indeed, it is! Fixed!